perun-core/src/main/java/cz/metacentrum/perun/core/api/OwnersManager.java

```diff
@@ -71,6 +71,20 @@
 	 */
 	Owner getOwnerById(PerunSession perunSession, int id) throws OwnerNotExistsException, InternalErrorException, PrivilegeException;
 
+	/**
+	 * Find owner by name.
+	 *
+	 * @param perunSession perun session
+	 * @param name name of the owner
+	 *
+	 * @return Owner with specified name
+	 *
+	 * @throws OwnerNotExistsException if owner with given name does not exist
+	 * @throws InternalErrorException
+	 * @throws PrivilegeException if user does not have sufficient permissions
+	 */
+	Owner getOwnerByName(PerunSession perunSession, String name) throws OwnerNotExistsException, InternalErrorException, PrivilegeException;
```

Comment:
We don't want to mention ``InternalErrorException`` in method definition and javadoc anymore, since its runtime.